conans/client/tools/system_pm.py

```diff
@@ -53,13 +66,33 @@ def install(self, packages, update=True, force=False):
             Get the system package tool install command.
         '"""
         packages = [packages] if isinstance(packages, str) else list(packages)
+
+        mode = self._get_sysrequire_mode()
+
+        if mode in ("verify", "disabled"):
+            # Report to output packages need to be installed
+            if mode == "disabled":
+                _global_output.error("The following packages need to be installed:\n %s" % "\n".join(packages))
```

Comment:
Is this an error? In this mode, users are assuming that those packages will be installed in the machine (otherwise the packages will fail). I'd say this is normal info, not error.

Reply:
Yes, that was my fault, I meant an error only if verify and not existing.